app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt

```diff
@@ -51,6 +51,12 @@ class PlayerActivity : AppCompatActivity() {
      */
     private val hideUnlockButtonAction = Runnable { unlockScreenButton.isVisible = false }
 
+    /**
+     * Runnable that hides [playerView] controller
+     */
+    private val hidePlayerViewControllerAction = Runnable { playerView.hideController() }
+
+
```

Comment:
```suggestion

```

Reply:
I should've been more clear on this one; there were two empty lines and it should've been just one. You now removed both so you need to add one back.

Reply:
Oh. I didn't see what this was for. I just committed all the suggestions on Github and squashed the commit.